Code/GraphMol/Descriptors/MolSurf.cpp

```diff
@@ -357,5 +361,23 @@ std::vector<double> calcPEOE_VSA(const ROMol &mol, std::vector<double> *bins,
 
   return res;
 }
+
+std::vector<double> calcCustomProp_VSA(const ROMol &mol, const std::string &customPropName,
+		                         std::vector<double> *bins, bool force) {
+  std::vector<double> lbins;
+  lbins.resize(bins->size());
+  std::copy(bins->begin(), bins->end(), lbins.begin());
+  std::vector<double> res(lbins.size() + 1, 0);
+
+  std::vector<double> vsaContribs(mol.getNumAtoms());
+  double tmp;
+  getLabuteAtomContribs(mol, vsaContribs, tmp, true, force);
+
+  std::vector<double> prop(mol.getNumAtoms(), 0.0);
+  prop = moldata3D.GetCustomAtomProp(mol, customPropName);
+  assignContribsToBins(vsaContribs, prop, lbins, res);
```

Comment:
I think assignContribsToBins is using "lbins" in a const way, if this is so, we don't need to copy bins at all, but we may need to change the assignContribsToBins signature (it's not exposed so I think this is ok)